experimental/dds/tree2/src/test/shared-tree/editing.spec.ts

```diff
@@ -810,6 +811,78 @@ describe("Editing", () => {
 			unsubscribe();
 		});
 
+		it("change, undo, childchange rebased over childchange", () => {
```

Comment:
I left some test cases around in this file for sequence field which were interesting for optional field (in that at some point I had an implementation which failed the test case, and I wondered how sequence field got it right).

I'm ok removing them or shifting to a different PR if we care.

Reply:
Happy to keep them as part of this PR.

Reply:
Maybe rename to this though:
```suggestion
		it("delete, undo, childchange rebased over childchange", () => {
```